src/test/unit/services/country.spec.js

```
@@ -0,0 +1,56 @@
+      axios.get.mockResolvedValue({ data: mockCountries });
+
+      const result = await countryService.getCountries();
+
+      expect(axios.get).toHaveBeenCalledWith('https://api.countrystatecity.in/v1/countries', {
+        headers: { 'X-CSCAPI-KEY': process.env.COUNTRY_API_KEY }
+      });
+      expect(result).toEqual(mockCountries);
+    });
+
+    it('should throw an error if the external API fails', async () => {
+        axios.get.mockRejectedValueOnce(new Error('Error fetching countries'));
```

ESLint

[error] Expected indentation of 6 spaces but found 8.

(indent)